app/models/admin_setting.rb

```diff
@@ -38,14 +39,17 @@ class << self
   end
 
   def self.default_skin
-    settings = current
-    if settings.default_skin_id.present?
-      Rails.cache.fetch("admin_default_skin") { settings.default_skin }
+    if self.first
+      self.first.default_skin_id ? self.first.default_skin : Rails.cache.fetch(default_skin_cache_key) { Skin.default }
     else
-      Skin.default
+      Rails.cache.fetch(default_skin_cache_key) { Skin.default }
     end
   end
 
+  def self.stats_updated_at
```

Comment:
I think this is already handled by the call to `delegate` above:
https://github.com/otwcode/otwarchive/blob/9dba531c689f9bcc1eac1a9e4ac50227bb0414e2/app/models/admin_setting.rb#L37-L39